packages/@adobe/spectrum-css-temp/components/tabs/index.css

```diff
@@ -31,6 +46,46 @@ governing permissions and limitations under the License.
   vertical-align: top;
 
   outline: none;
+
+  &.spectrum-Tabs--isCollapsed {
+    inset-inline-start: 0;
+    height: var(--spectrum-tabs-item-height);
+
+    /* FieldButton Picker focus ring override */
+    & button {
+      &::before {
```

Comment:
Updated the focus ring for the collapsed Tab picker. It now has spacing consistent to the focus ring for the tab items, but there weren't specific designs in the Tabs XD file so this is a tentative change